core-services/prow/02_config/kiegroup/kie-cloud-tests-container/_pluginconfig.yaml

```diff
@@ -0,0 +1,28 @@
+plugins:
+  kiegroup/kie-cloud-tests-container:
```

Comment:
Does PQE want all of these plugins on their repository? I usually only see one or two (typically `trigger`). I think it's fine to keep them, just don't want to bombared their repository with bot and automation functionality that they may not want.

Reply:
Okay @calebevans , I will ask PQE to change this. We added because of some failures during the run.

Reply:
@Sau1506mya no worries, you can add those other plugins if you want. But, the only one you need for our purposes is the `trigger` one. Here is an example of the minimal config: https://github.com/openshift/release/blob/990a45fac200616b11ba55096b0392f918295a6c/core-services/prow/02_config/windup/windup-ui-tests/_pluginconfig.yaml